pbf_file_size_estimation/__init__.py

```diff
@@ -1,2 +1,2 @@
-__version__ = '0.0.7'
+__version__ = '1.0.0'
```

Comment:
> we haven't documented that setting - not public API ;-)

I found this typo because the constant is being [imported and used](https://github.com/geometalab/osmaxx/blob/269cfaab5464510383609bed167780b783834bae/osmaxx/conversion/management/commands/result_harvester.py#L100) in a project using this django app, so ... :stuck_out_tongue:

> I don't like 1.0.0 releases on a rather untested project, especially when they are on pypi.

I thought we agreed on [semantic versioning](http://semver.org/) and that semantic versioning is not uncommon on PyPI. If we want to do both, semantic versioning _and_ version numbers indicating the (non-)ripeness of the project, we must turn the change into a backwards-compatible one. (Which isn't too hard: Just alias the new constant with the old one.)